import-export-cli/docs/apictl_mi_add_user.md

```diff
@@ -21,7 +21,7 @@ NOTE: The flag (--environment (-e)) is mandatory
 ### Options
 
 ```
-  -e, --environment string   Environment to be searched
+  -e, --environment string   Environment of which micro integrator the user should be added
```

Comment:
Shall we change the description as 
"Environment of the micro integrator to which a new user should be added" or "The micro integrator environment to which a new uer should be added"? (So that it would be more meaningful. Also, change the descriptions for loggers etc as well like above)

Reply:
Fixed with f2471fb